### ISSUE TITLE
.pol can not see windows settings/security settings

### ISSUE THREAD

**Issue by scerazy** [Fleex255/PolicyPlus#20]
.pol opens fine, but can not see what is configured in GPO under Policies/Windows settings/Security settings

**Comment by Fleex255** [comment]
Indeed, Policy Plus is currently designed only for ADMX-guided policy settings, i.e. those under the Administrative Templates branch. UI support for other kinds of policies is probably a long way off, but for what it's worth, the corresponding Registry entries are shown in Edit Raw POL. It might be possible for someone to write ADMX files that correspond to a few of the Security Settings, especially Security Options.

**Comment by Lewis8379** [comment]
Related question, if I made some edits in PolicyPlus, and some Security Settings edits in the standard Group Policy software, would exporting and importing to another computer using PolicyPlus also copy those Security Settings edits?

**Comment by Fleex255** [comment]
Yes, if:

* The Security Settings you changed are Registry-based (you can check whether *File | Edit Raw POL* shows entries with relevant-looking value names), **and**
* You export and import the policies as a POL file (rather than Semantic Policy, which can only express ADMX-guided policies)

**Comment by Lewis8379** [comment]
Thanks for the reply. I've just tried changing "Enforce password history" under Computer > Windows > Security > Account > Password, which was a random choice, from 0 to 1. This is not shown under Edit Raw POL > Computer, which is blank. 

So in these cases where they are not Administrative Templates, or Registry based, is it not possible to copy/export these policies across at all? Thanks.

**Comment by Fleex255** [comment]
Unfortunately Policy Plus cannot currently handle policies that are not POL/Registry-based, sorry.

However, there may be other tools that can work with the security policies. While I have not tried it myself, Microsoft's [`secedit`](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/secedit) utility looks promising.
